extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -131,6 +133,14 @@ export class SchemaCompareResult {
 			this.sourceTargetFlexLayout.addItem(this.targetNameComponent, { CSSStyles: { 'width': '45%', 'height': '25px', 'margin-top': '10px', 'margin-left': '15px' } });
 
 			this.loader = view.modelBuilder.loadingComponent().component();
+			this.waitText = view.modelBuilder.text().withProperties({
+				value: localize('schemaCompare.WaitText', 'Initializing Comparision. This might take a moment.')
+			}).component();
+
+			this.startText = view.modelBuilder.text().withProperties({
+				value: localize('schemaCompare.StartText', 'Press Compare to start Schema Comparision')
```

Comment:
Aren't you changing this to "OK" in this PR?

Reply:
I guess not? What's the OK button for then?

Reply:
The ok button is before this page. The behavior now is that - when user just selects source and target and press "ok" (which was compare before) in the dialog it doesn't not start comparison right away but opens the compare page so that user can select options and so on and then use "Compare" button to start comparison.